Resources/Prototypes/Traits/speech.yml

```
@@ -7,7 +7,9 @@
   name: trait-accentless-name
   description: trait-accentless-desc
   category: SpeechTraits
-  cost: 2
+  # Moffstation - Begin - change cost to 1 for point balancing
+  cost: 0
+  # Moffstation - End
```

Only a one line guard comment needed here
```suggestion
cost: 0 # Moffstation - change cost to 0 for point balancing
```